src/Sarif/Schemata/sarif-schema.json

```diff
@@ -367,6 +367,30 @@
       "required": [ "threadFlows" ]
     },
 
+    "configurationOverride": {
+      "description": "Information about how a specific tool report was reconfigured at runtime.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+
+        "configuration": {
+          "description": "Specifies how the report was configured during the scan.",
+          "$ref": "#/definitions/reportingConfiguration"
+        },
+
+        "descriptor": {
+          "description": "A reference used to locate the descriptor relevant to this configuration override.",
```

Comment:
>relevant to this configuration override [](start = 68, length = 39)

"relevant to this configuration override" => "whose configuration is overridden"